### DISCUSSION TITLE
How to Restore Toolbox

### DISCUSSION THREAD

**Discussion by linhao231035** [xournalpp/xournalpp#5049] [Q&A]
it is my first time to use the xournalpp,I did a wrong thing to make the screen look like this
![屏幕](https://github.com/xournalpp/xournalpp/assets/96814455/6f9bc831-948e-4ce4-a321-68c2edf04842)
How to restore it to the way it was just installed and opened? Thank you

**Comment by rolandlo** [comment]
Hitting `F5` once or twice should probably do it. Maybe you will also have to select a toolbar under menu `View > Toolbars`.

**Comment by linhao231035** [comment]
ok,i got it，thanks
